internal/transfers/repository.go

```diff
@@ -60,15 +60,17 @@ func (r *SQLRepo) Close() error {
 	return r.db.Close()
 }
 
-func (r *SQLRepo) getUserTransfers(userID id.User) ([]*model.Transfer, error) {
-	query := `select transfer_id from transfers where user_id = ? and deleted_at is null`
+func (r *SQLRepo) getUserTransfers(userID id.User, params transferFilterParams) ([]*model.Transfer, error) {
+	query := `select transfer_id from transfers
+where user_id = ? and created_at >= ? and created_at <= ? and deleted_at is null
+order by created_at desc limit ? offset ?;`
 	stmt, err := r.db.Prepare(query)
 	if err != nil {
 		return nil, err
 	}
 	defer stmt.Close()
 
-	rows, err := stmt.Query(userID)
+	rows, err := stmt.Query(userID, params.StartDate, params.EndDate, params.Limit, params.Offset)
```

Comment:
Before we query is `params.EndDate` `23:59:999.000` ? I could have missed it but didn't see anything below.

Reply:
It's `now() + "24 hours"` basically. We could bump it forward more. 

```go
EndDate:   time.Now().Add(24 * time.Hour),
```

In the future we need to support #447 (scheduled transfers) where I think this endpoint should order by scheduled_at or created_at (coalesce the two) because transfers without scheduled_at are to be uploaded/posted on the current day.